### PR TITLE
mock net.InterfaceAddrs to make its unit test become available

### PR DESCRIPTION
**Pull request by zhlsunshine**
**Please provide a description of this PR:**
mock net.InterfaceAddrs to make its unit test become available

### CODE REVIEW

File: tools/istio-iptables/pkg/cmd/root_test.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+					net.ParseIP("127.0.0.1"),
+					net.ParseIP("1.2.3.5"),
+					net.ParseIP("2222:3333::1"),
+				})
+			},
+			expected: false,
+		},
+	}
+	for _, tt := range tests {
+		LocalIPAddrs = tt.lipas
```
[the remark this answers is not in the document]
Comment (zhlsunshine) [reply tools/istio-iptables/pkg/cmd/root_test.go]: sure

[the remark this answers is not in the document]
Comment (zhlsunshine) [reply tools/istio-iptables/pkg/cmd/root_test.go]: done

[the remark this answers is not in the document]
Comment (zhlsunshine) [reply tools/istio-iptables/pkg/cmd/root_test.go]: Hi @howardjohn, could you please help to approve the PR if it's okay? Thanks! ^_^

**Review by ramaraochavali — APPROVED** [review]

**Pull request merged**